src/app.json.template

```diff
@@ -28,5 +28,6 @@
         "linkedin": {
             "clientId": "[ CLIENTID HERE ]"
         }
-    }
+    },
+    "defaultGridState": "grid-view"
```

Comment:
I am assuming you mean that I am changing the default from list-view **to** grid-view.
No this was not intentional. I will be changing the default back to list-view unless Mitesh deems otherwise.